armv7m7-imxrt106x/low.c

```diff
@@ -317,15 +317,6 @@ int low_launch(void)
 {
 	syspage_save();
 
-	/* Give the LPUART transmitters some time */
-	timer_wait(100, TIMER_EXPIRE, NULL, 0);
-
-	/* Tidy up */
-	serial_done();
-	timer_done();
-
-	low_done();
-
```

Comment:
My mistake, I think it would be better to leave this as is, remove changes from cmd_go and make low_launch in imxrt117x analogous to this one.

Reply:
Sounds good.